bower.json

```diff
@@ -29,7 +29,8 @@
     "px-actionable-design": "^2.0.5",
     "px-popover": "^2.1.2",
     "px-slider": "^2.1.1",
-    "px-rangepicker": "^3.2.0"
+    "px-rangepicker": "^3.1.0",
```

Comment:
This would revert back to old version?

Reply:
No, `^3.1.0`,  and `^3.1.0` are basically the same and mean `3.x.x`.

https://stackoverflow.com/a/22345808/1331425

But yeah, this change is unnecessary.